src/applications/static-pages/sharable-links/sharableLink.js

```diff
@@ -162,24 +163,22 @@ const SharableLink = ({ dataEntityId, idx, showSharableLink }) => {
             className="usa-button-unstyled"
             aria-label={`Copy ${dataEntityId} sharable link`}
             id={`button-${dataEntityId}`}
-            onBlur={() => {
-              onBlur(dataEntityId);
-            }}
-            onFocus={() => {
+            onClick={event => {
+              event.persist();
+              if (!event || !event.target) return;
+              displayFeedback(event.target);
+              copyToUsersClipBoard(dataEntityId, event.target);
               onFocus(dataEntityId);
             }}
           >
             <ShareIcon
               aria-hidden="true"
               className={`fas fa-link sharable-link`}
               feedbackActive={feedbackActive}
-              onClick={event => {
-                event.persist();
-                if (!event || !event.target) return;
-                displayFeedback(event.target);
-                copyToUsersClipBoard(dataEntityId, event.target.parentElement);
-              }}
               id={`icon-${dataEntityId}`}
+              onBlur={() => {
```

Comment:
It looks like this is just a styled `<i>`, if that's the case, this blur event won't trigger since it isn't a focusable element

Reply:
ah, that makes sense, thanks! I'll remove it.